.github/workflows/privacy.yml

```diff
@@ -1,10 +1,20 @@
 name: Privacy Tests
 
 on:
-  pull_request:
+  pull_request_target:
     types: [ labeled ]
+
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+    paths:
+      - 'node_modules/@duckduckgo/content-scope-scripts/**'
```

Comment:
This would allow Android code to break the implementation also, I don't think we should limit this tbh.